stories/adminbar.stories.js

```diff
@@ -42,6 +43,7 @@ import CollectModuleData from '../assets/js/components/data/collect-module-data'
 storiesOf( 'Global', module )
 	.add( 'Admin Bar', () => {
 		global._googlesitekitLegacyData = wpAdminBarData;
+		disableFeature( 'widgets.adminBar' );
```

Comment:
This should be added to `resetFeatures` in `.storybook/config` – stories should only enable features where needed. Features are then opt-in per-story, never opt-out.

Reply:
Updated @aaemnnosttv